frontend/.nvmrc

```diff
@@ -0,0 +1 @@
+lts/gallium
```

Comment:
Is this a correct entry?
I expected a version-number instaed of a path-string...

Reply:
Like I write, I copy the version from docker. 
The Frontend Docker use node:16-alpine3.14 which is the latest version from 16 named gallium, so this string does the same. Tell nvm to use the latest release from gallium.